light/provider/http/http.go

```diff
@@ -4,20 +4,20 @@ import (
 	"context"
 	"fmt"
 	"math/rand"
-	"regexp"
 	"strings"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
I think we should use the stdlib error wrapping [introduced with Go 1.13](https://blog.golang.org/go1.13-errors), e.g. `fmt.Errorf´ with `%w`.